README.md

```diff
@@ -1,36 +1,66 @@
-# Civic Data Ecosystem website
```

Comment:
replace this with a brief readme about the site. 
- installation notes
- credit to theme and link to it